src/main.rs

```diff
@@ -424,18 +424,18 @@ async fn get_disk_usage() -> String {
 
 #[tokio::main]
 async fn main() {
-    let hostname_fut = spawn(get_hostname());
-    let greeting_fut = spawn(greeting());
-    let datetime_fut = spawn(get_datetime());
-    let weather_fut = spawn(get_weather());
-    let release_fut = spawn(get_release());
-    let kernel_fut = spawn(get_kernel());
-    let memory_fut = spawn(get_memory());
-    let disk_fut = spawn(get_disk_usage());
-    let environment_fut = spawn(get_environment());
+    let hostname_fut = spawn_blocking(get_hostname);
```

Comment:
these all changed back to `spawn_blocking` because they block on the output of commands